bin/flash.flashair

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+# It is outside the scope of this script to ensure that the card is online
+UL_DIR=`mktemp -d`
+
+# Run this script to copy files as they must appear on the target into
+# the given directory.
+. "${bin_dir}/${flashair_copy_script}"
+
+if [ ! -z "${flashair_rmlist}" ]; then
+  ARGS="rmlist:${flashair_rmlist}"
+fi
+
+ARGS="${ARGS} push:${UL_DIR}"
+push-flashair.py ${flashair_ip} ${ARGS}
```

Comment:
OK, I suppose that's reasonable, but that means poweron.* here need to do the ping -c1 stuff. poweron.* would either (a) need a hook to wait for a list of resources, or perhaps u-boot-power-on should call a Bash hook function that's defined in bin/${hostname}/conf.${board_type}_${board_instance} to do the waiting (which could in turn call out to some common function).

I suppose that can be a separate commit to this one though.

As background, take a look at https://github.com/swarren/u-boot-ci-scripts/blob/master/test.sh; that executes u-boot-test-power-on (from this repo), runs ./src/u-boot/test/py/test.py, then executes u-boot-test-power-off.